src/targets/Jumper_2400.ini

```diff
@@ -0,0 +1,19 @@
+
+# ********************************
+# Transmitter targets
+# ********************************
+
+[env:AION_2400_T-Pro_TX_via_WIFI]
```

Comment:
This should be `Jumper_AION_2400_T-Pro_TX_via_WIFI` to match the format of all the other vendor targets.